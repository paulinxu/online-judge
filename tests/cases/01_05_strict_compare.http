###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 411
# content-type: application/json
# date: Thu, 25 Jul 2024 15:33:02 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Wrong Answer","time":0}],"created_time":"2024-07-25T15:33:03.369995772Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0},"updated_time":"2024-07-25T15:33:03.576772328Z"}
