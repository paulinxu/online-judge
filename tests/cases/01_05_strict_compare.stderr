[2024-07-25T15:33:02Z INFO  actix_server::builder] starting 16 workers
[2024-07-25T15:33:02Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2024-07-25T15:33:03Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000108
[2024-07-25T15:33:03Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 411 "-" "-" 0.207188
